M2/Macaulay2/e/fplll-interface.cpp

```diff
@@ -3,7 +3,13 @@
 
 #ifdef HAVE_FPLLL
```

Comment:
Do we need this file?

Reply:
@mikestillman -- this is for you

Reply:
Currently, we do not use fplll, and it is not on the test path, and the version of fplll we refer to is relatively old.  However, it might be worth it to get this going again.  LLL (and BKZ) are important algorithms.  I would want to see how this compares with the new flint LLL, BKZ code, and with NTL (currently, our LLL is using NTL).  However, it might be that one of these others is better.